### ISSUE TITLE
Request screen via webAPI

### ISSUE THREAD

**Issue by Borgso** [nibogd/ingress-ice#62]
Would it be possible to run ingress-ice as a webapp and then request a screen via iehttp://host:8080/grab?lat=0.0&lon=0.0&z=17 
This would make it possible to have it in background and use favorite script to generate whatever you want..

**Comment by nibogd** [comment]
I like the idea and it's theoretically possible. But I don't have much free time, so it won't be fast, sorry.

**Comment by rawdr** [comment]
Agreed... if it could host a simple web page which 1) allowed starting/stopping, 2) allowed changing of parameters (url, pixels, interval, etc), and 3) listed resulting png's in directory....

then it would be pretty easy to use solely from a mobile phone from the field.

**Comment by h3tz** [comment]
intelrecorder.com ?
